src/sentry/conf/urls.py

```diff
@@ -1,12 +1,3 @@
-"""
-sentry.conf.urls
-~~~~~~~~~~~~~~~~
-
-These are additional urls used by the Sentry-provided web server
-
-:copyright: (c) 2012 by the Sentry Team, see AUTHORS for more details.
```

Comment:
Let me quote the PR title:
>ref: revise setup.py

This file says `src/sentry/conf/urls.py` - I sense contradictions in you but the force is strong.

Reply:
scope creep gets the best of us, haha